__tests__/workers/postUpdated.ts

```diff
@@ -459,7 +459,14 @@ it('should save a new post with content curation', async () => {
   });
   const posts = await con.getRepository(Post).find();
   expect(posts.length).toEqual(3);
-  expect(posts[2].contentCuration).toStrictEqual(['news', 'story', 'release']);
+  const curatedPost = posts.find(
+    (item) => item.yggdrasilId === 'f99a445f-e2fb-48e8-959c-e02a17f5e816',
+  );
+  expect(curatedPost!.contentCuration).toStrictEqual([
+    'news',
+    'story',
+    'release',
+  ]);
```

Comment:
This is flaky and kept failing while I was fixing the issue so fixed it as well. Basically db does not guarantee order of rows.

Reply:
I fixed it on one of my PRs, you can have a look

Reply:
I'll revert then.

Reply:
I also fixed that it one of my PRs 😬

Reply:
In #1603, running the tests 2 more times to see that nothing else is flaky with psql15